app/templates/app/_base.html

```diff
@@ -35,15 +35,21 @@
          <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
           integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+<!--    Google Handrwriting Font-->
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Indie+Flower">
 <!--    axios -->
 <!--    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>-->
     <!-- our native styles -->
     <link rel="stylesheet" href="{% static 'app/styles.css' %}">
+
    <!-- Import Vue.js -->
-    <script src="https://unpkg.com/vue@next"></script>
+    <script src="https://unpkg.com/vue@3.0.0-beta.12/dist/vue.global.js"></script>
```

Comment:
Do we still need to load Vue if we're installing from npm?

Reply:
deleted